3209.json

```diff
@@ -0,0 +1,93 @@
+{
+"None" : "",
+"SysIntermission_GameWillStart" : "Spiel Startet in 5 sekunden!",
+"SysEndgame_GameOver" : "Verloren!",
+"SysEndgame_WinnerNone" : "Kein Gewinner!!",
+"SysEndgame_WinnerSingle" : "Der gewinner ist... {0}!",
+"SysEndgame_WinnerTwo" : "Die Beiden Gewinner sind...: {0} and {1}!",
+"SysEndgame_WinnerMoreThanTwo" : "Die Gewinner sind...: {0}!",
+"SysEndgame_Thanks" : " Vielen dank fürs spielen!\njeden monat kommt neues Zeug.",
+"SysIntermission_HitToStart" : "Mach die Rote Box Kalt indem du sie {0} mal schlägst! damit \nDamit startest du das spiel!",
+"SysDesyncError" : "(Nächstes spiel startet...)",
+"PlaySuccess" : "du hast es gut gemacht!",
+"PlayFailure" : "du hast verkackt.",
```

Comment:
Please check this line and the others below to make sure that all sentences start with an uppercase letter

Can the sentence end with an exclamation mark, to remain consistent with the rest of the game lines?